tests/router_test.ts

```diff
@@ -1500,6 +1500,67 @@ scenarios.forEach(function(scenario) {
       });
   });
 
+  test('abort query param only', function(assert) {
+    assert.expect(6);
```

Comment:
There are 8 `assert.*` invocations (excluding the `assert.ok(false` one), why is this number 6? Is there an extra branch with assertions that we don't hit?

Reply:
@richgt - As you resolve this, can you add assertion messages for each of the assertions that run (it will help figure out which are *not* running).

Reply:
yeah, got rid of the one path that wasn't hit.

Reply:
added assertion messages